aws_lambda_script/app/resources.py

```diff
@@ -48,14 +66,31 @@ def get_user_email(args):
         logger.exception("Error verifying token: %s", error)
         abort(401, description="Not authorized")
 
+def is_auth_user_in_admin_group():
```

Comment:
Is it possible to refactor and get all the information we need from the token at once?

eg 

`def get(self):
    args = parser.parse_args()
    owner_email = get_user_email(args)
    user_groups = get_user_attributes(args).get("cognito:groups", [])
 `

then return the email and groups rather than two separate functions for emails and groups as it is the same token?